core/domain/email_manager.py

```diff
@@ -508,3 +510,36 @@ def send_instant_feedback_message_email(
             recipient_id, feconf.SYSTEM_COMMITTER_ID,
             feconf.EMAIL_INTENT_FEEDBACK_MESSAGE_NOTIFICATION, email_subject,
             email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_flag_exploration_email(
+        exploration_title, exploration_id, reporter_id, report_text):
+    email_subject = 'Exploration flagged by user: "%s"' % exploration_title
+
+    email_body_template = (
+        'Hello Moderator,<br>'
+        '%s has submitted a new report on the exploration %s on '
```

Comment:
Looking at this again, I think that `%s has flagged exploration "%s" on the following grounds:` is a better wording.